lib/stimulus_reflex/reflex.rb

```diff
@@ -129,7 +129,13 @@ def params
     @_params ||= ActionController::Parameters.new(request.parameters)
   end
 
-  def dom_id(record_or_class, prefix = nil)
-    "#" + ActionView::RecordIdentifier.dom_id(record_or_class, prefix).to_s
+  def dom_id(record, prefix = nil)
+    return "#" + record.model_name.plural if record.is_a?(ActiveRecord::Relation)
+    return "#" + ActionView::RecordIdentifier.dom_id(record, prefix).to_s if record.is_a?(ActiveRecord::Base)
+    [record.to_s, prefix].compact.join("_")
+  end
+
+  def wrap(content, resource)
+    "<div id=\"#{dom_id(resource)}\">#{content}</div>"
```

Comment:
Small change, but imo it somehow looks cleaner than interpolating variables into the string. 
```suggestion
    tag.div content, id: dom_id(resource)
```

(maybe needs an include for `ActionView::Helpers::TagHelper`)

Reply:
This was a great suggestion. Thanks!